File: fixtures/python_output/post_with_extra_whitespace.py
import requests

headers = {
    'accept': 'application/json',
    'Content-Type': 'multipart/form-data',
}

files = {
    'files': ('47.htz', open('47.htz', 'rb')),
    'name': (None, '47'),
    'oldMediaId': (None, '47'),
    'updateInLayouts': (None, '1'),
    'deleteOldRevisions': (None, '1'),
}

response = requests.post('http://129.12.19.62/api/library', headers=headers, files=files)

File: test/fixtures/php-guzzle/post_with_extra_whitespace.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;
use GuzzleHttp\Psr7;

$client = new Client();

$response = $client->post('http://localhost:28139/api/library', [
    'headers' => [
        'accept'       => 'application/json',
        'Content-Type' => 'multipart/form-data'
    ],
    'multipart' => [
        [
            'name' => 'files',
            'contents' => Psr7\Utils::tryFopen('47.htz', 'r')
        ],
        [
            'name' => 'name',
            'contents' => '47'
        ],
        [
            'name' => 'oldMediaId',
            'contents' => '47'
        ],
        [
            'name' => 'updateInLayouts',
            'contents' => '1'
        ],
        [
            'name' => 'deleteOldRevisions',
            'contents' => '1'
        ]
    ]
]);
